dvc/cache.py

```diff
@@ -87,5 +88,5 @@ def _get_remote(self, config, name):
         if not remote:
             return None
 
-        settings = self.repo.config.get_remote_settings(remote)
+        settings = get_remote_settings(self.repo.config, remote)
```

Comment:
does it make sense to just pass `remote` to the Remote itself? Remote can accept both options if it makes sense to accept settings as part of the API.